### ISSUE TITLE
focus mode cannot be shown with NVDA 2014.4 or prior versions

### ISSUE THREAD

**Issue by nishimotz** [nvdajp/focusHighlight#1]
Due to my errata, ability to indicate the focus mode is applicable only with NVDA 2015.1 or later versions.
